bindings/ocaml/dune

```diff
@@ -0,0 +1,12 @@
+(library
+  (name xcp_ng_vdi_stream)
+  (public_name xcp_ng_vdi_stream)
+  (c_library_flags -lpthread)
+  (self_build_stubs_archive (xcp_ng_vdi_stream))
+)
+
+(rule
+  (targets libxcp_ng_vdi_stream_stubs.a dllxcp_ng_vdi_stream_stubs.so)
```

Comment:
In theory yes, but in practice there are many problems:
- We can have a `dune.in` file and a call to `configure_file` but in this case, it's necessary to copy all OCaml sources in the build dir.
- We can't build an opam package at project root because a dune file must be accessible in sources.

Reply:
Okay, in this case I think its better to duplicate the name too.